tests/unit/test-cmd/test.run.js

```diff
@@ -347,42 +409,11 @@ describe('run', () => {
       // Simulate executing the handler when a source file changes.
       return callArgs.onChange()
         .then(() => {
-          assert.equal(config.client.reloadAddon.called, true);
-          const reloadArgs = config.client.reloadAddon.firstCall.args;
+          assert.equal(config.addonReload.called, true);
+          const reloadArgs = config.addonReload.firstCall.args;
           assert.ok(config.addonId);
-          assert.equal(reloadArgs[0], config.addonId);
-        });
-    });
-
-    it('notifies user on error from source change handler', () => {
```

Comment:
If I'm not wrong, once we revert the changes related to the `addonReload` helper, these changes to the tests are not needed anymore and they can also be reverted (and the resulting change should be shorter and simpler)